runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -41,11 +41,14 @@
 #include "jitregmap.h"
 #include "j2sever.h"
 #include "vmaccess.h"
+#include "objhelp.h"
```

Comment:
Why has this been added?

Reply:
Some declarations came via `MHInterpreter.hpp` which may not be included with this change.